test/body-parser.js

```diff
@@ -51,6 +58,31 @@ describe('bodyParser()', function () {
       .expect(200, '{"user":"tobi"}', done)
   })
 
+  if (testAsyncHooks) {
```

Comment:
Instead of just leaving out the test in silence, please make it skip the test if it cannot run so the test output shows the test is there.